tools/rosmaster/src/rosmaster/master_api.py

```diff
@@ -236,13 +234,13 @@ class ROSMasterHandler(object):
     this parameter (see ros::client::getMaster()).
     """
     
-    def __init__(self):
+    def __init__(self, nwork):

```

Comment:
This is public API which can't be changed in a breaking way like this. Please make the new arguments optional.